Sprout/View Controllers/ComposePostViewController.m

```diff
@@ -0,0 +1,45 @@
+//
+//  ComposePostViewController.m
+//  Sprout
+//
+//  Created by Sarah Wang on 7/19/21.
+//
+
+#import "ComposePostViewController.h"
+#import <QuartzCore/QuartzCore.h>
+#import "Post.h"
+#import "Task.h"
+
+@interface ComposePostViewController ()
+@property (weak, nonatomic) IBOutlet UITextView *composeTextView;
+@property (nonatomic, strong) PFUser *user;
+
+@end
```

Comment:
nit: Would expect this to be newline between interface and end or no newlines. Consistency is more important than the specific pattern.